openmc/mesh.py

```diff
@@ -1204,22 +1215,27 @@ class CylindricalMesh(StructuredMesh):
         The default value is [0, 2π], i.e. the full phi range.
     z_grid : numpy.ndarray
         1-D array of mesh boundary points along the z-axis.
-    origin : numpy.ndarray
-        1-D array of length 3 the (x,y,z) origin of the mesh in
-        cartesian coordinates
     indices : Iterable of tuple
         An iterable of mesh indices for each mesh element, e.g. [(1, 1, 1),
         (2, 1, 1), ...]
 
     """
 
-    def __init__(self, mesh_id: int = None, name: str = ''):
+    def __init__(
+        self,
+        mesh_id: int = None,
+        name: str = '',
+        origin: typing.Tuple[float] = (0., 0., 0.),
```

Comment:
Just to mention I've added a ```Tuple``` type hint to origin as that agrees with the default value used. However the doc string asks for an array and I notice we change it to an array in the setter. Not sure what is preferable here but I guess these are the sorts of things we uncover while adding type hints

Reply:
I think it’s preferable to have the type hint be an iterable of float or real if that’s possible and for the internal representations and what you get back from getters to be numpy arrays